_data/team-members.yml

```diff
@@ -79,6 +79,21 @@ members:
        - title: dotty
          percentage: 10
     active: true
+  - member: alexandre
+    name: Alexandre Archambault
+    title: Software Development
+    email: alexandre.archambault@epfl.ch
+    github: alexarchambault
+    twitter: alxarchambault
+    photo:
```

Comment:
No photo? :cry:

Reply:
Got to find a photo for ✨professional✨ contexts...

Reply:
OK, let's merge it as is for now, but I'll keep poking you until you've added a picture :)